CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_shopify_source v0.18.1
+
+[PR #103](https://github.com/fivetran/dbt_shopify_source/pull/103) includes the following updates:
+
+## Under the Hood
+- Added BigQuery JSON field support for the following model and columns:
+  - `stg_shopify__order`: `total_shipping_price_set` column
+  - `stg_shopify__transaction`: `receipt` column
+  - `stg_shopify__collection`: `rules` column
+- Introduced new the new `json_to_string()` macro to be used in the BigQuery JSON field support.
+- Included json versions to the integration tests to ensure json data type compatibility.
+
```

Comment:
Looks like we also updated the pull request template, should we call that out.

Reply:
Sure we can. I don't think it's all that necessary, but wouldn't hurt to include.